### ISSUE TITLE
LIG creation issue in Non-redundant mode

### ISSUE THREAD

**Issue by fedorovant** [HewlettPackard/POSH-HPOneView#240]
Please fill in as much information as possible to help resolve your issue.

### Expected Behavior

Create two separate LIG for Ethernet Side-A only in one Frame configuration  (slot 3) and Side-B only (slot 6 on the same frame).

### Actual Behavior

#---Create LIG----Here is for 1 frame A(Non-Redundant)
New-HPOVLogicalInterconnectGroup -Name 'LIG-1' -FrameCount 1 -InterconnectBaySet 3 -FabricModuleType 'SEVC40F8' -Bays @{Frame1 = @{Bay3 = 'SEVC40f8'}} -FabricRedundancy 'Aside'

New-HPOVLogicalInterconnectGroup : The JSON sent in the request contained a
String where a "redundancyType" was required on line 1 near column 1,063.
Correct the content of the JSON and try the request again.
At line:1 char:1
+ New-HPOVLogicalInterconnectGroup -Name 'LIG-1' -FrameCount 1
-InterconnectBaySet ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
~~~
    + CategoryInfo          : InvalidOperation: (Send-HPOVRequest:String) [New
   -HPOVLogicalInterconnectGroup], InvalidOperationException
    + FullyQualifiedErrorId : InvalidAuthOperation,New-HPOVLogicalInterconnect   Group

### Steps to reproduce
Connect Synergy OV
Start command:
New-HPOVLogicalInterconnectGroup -Name 'LIG-1' -FrameCount 1 -InterconnectBaySet 3 -FabricModuleType 'SEVC40F8' -Bays @{Frame1 = @{Bay3 = 'SEVC40f8'}} -FabricRedundancy 'Aside'


### Version Information
HPE OneView PowerShell Library Version (`Get-HPOVVersion`): 3.0.1221.2746 
HPE OneView Appliance Version:  3.00.05-0270544
Output from `$PSVersionTable` on your Windows Host:

Name                           Value
----                           -----
PSVersion                      5.1.14393.693
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.14393.693
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1

**Comment by ChrisLynchHPE** [comment]
This is fixed in [Release 3.0.1264.2772](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v3.0.1264.2772).

**Comment by ChrisLynchHPE** [comment]
Closing issue due to no further activity.  Feel free to re-open the issue if it still remains.
